### ISSUE TITLE
Influxdb not starting

### ISSUE THREAD

**Issue by do-web** [rancher/rancher#12170]
Influxdb not running:

Error: influxdb not starting

**Rancher versions:**
rancher/server: latest
rancher/agent: v1.2.10

**Infrastructure Stack versions:**
healthcheck: v0.3.6
ipsec: rancher/net:holder
network-services: rancher/metadata:v0.10.2
scheduler: rancher/k8s:v1.9.4-rancher1-1
kubernetes (if applicable):

Client Version: version.Info{Major:"1", Minor:"9", GitVersion:"v1.9.0", GitCommit:"925c127ec6b946659ad0fd596fa959be43f0cc05", GitTreeState:"clean", BuildDate:"2017-12-15T21:07:38Z", GoVersion:"go1.9.2", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"9+", GitVersion:"v1.9.4-rancher1", GitCommit:"6668bdc8423b0cc7eae01080fcbddb1837652c74", GitTreeState:"clean", BuildDate:"2018-03-13T03:14:03Z", GoVersion:"go1.9.3", Compiler:"gc", Platform:"linux/amd64"}

**Docker version: (`docker version`,`docker info` preferred)**
Docker version 17.03.2-ce, build f5ec1e2

**Operating system and kernel: (`cat /etc/os-release`, `uname -r` preferred)**

4.9.0-6-amd64

**Type/provider of hosts: (VirtualBox/Bare-metal/AWS/GCE/DO)**

**Setup details: (single node rancher vs. HA rancher, internal DB vs. external DB)**
Kubernetes Template

**Environment Template: (Cattle/Kubernetes/Swarm/Mesos)**
Kubernetes Template

**Steps to Reproduce:**
Install Rancher
Setup a host with Kubernetes

**Results:**
Influxb not starting

 InfluxDB starting, version unknown, branch unknown, commit unknown
Go version go1.8.3, GOMAXPROCS set to 4
Using configuration at: /etc/config.toml
run: open server: listen: listen tcp 46.38.243.234:8088: bind: cannot assign requested address

46.38.243.234 seems to be a DNS

**Comment by superseb** [comment]
Usually caused by a misbehaving DNS server that resolves `localhost` to itself (wildcard or actual record). See https://github.com/kubernetes/heapster/issues/1804

Can you try the fix (setting a proper nameserver) and see if that solves it? If not, please report back and we can investigate.

**Comment by huanga** [comment]
I'm seeing this issue but the IP address in my prompt is within our office, and is not broadcasting as a DNS server. I've attempted to verify this by doing `dig google.com @ip.in.error.log` as well as `dig localhost @ip.in.error.log` and neither resolves. I've also tried to `dig localhost` on the host machine, which should perform the same lookup as the instance did, but get correctly `127.0.0.1`. Is there any recommended debugging approach to find the problem DNS server?
